core/model/modx/processors/workspace/packages/getattribute.class.php

```diff
@@ -39,8 +39,12 @@ public function initialize() {
     public function process() {
         $attributes = array();
         $attributesToGet = explode(',',$this->getProperty('attributes',''));
+        $Parsedown = new Parsedown();
         foreach ($attributesToGet as $attribute) {
-            $attributes[$attribute] = $this->transport->getAttribute($attribute);
+            $data = $this->transport->getAttribute($attribute);
+            $attributes[$attribute] = in_array($attribute, ['changelog', 'license', 'readme'])
+                ? $Parsedown->text(htmlentities($data, ENT_COMPAT, 'UTF-8'))
```

Comment:
I would add such bit mask `ENT_COMPAT | ENT_HTML5`

Reply:
Markdown don`t work after ENT_HTML5 
![screen shot 2018-04-11 at 11 52 47](https://user-images.githubusercontent.com/1257284/38609690-03046a88-3d7f-11e8-93ae-710e5f1edd29.png)

Reply:
MODX 3 requires php 5.6+. In this case it can be simplified to just htmlentities() because `ENT_COMPAT | ENT_HTML401` is default value. And since 5.6 encoding param use value of conf option `default_charset` by default. And option set as "UTF-8" by default.

More details by links:
http://php.net/manual/en/function.htmlentities.php
http://php.net/manual/en/ini.core.php#ini.default-charset

Reply:
facepalm.jpg

Okay, I will create another commit.